package-lock.json

```diff
@@ -10834,11 +10834,6 @@
           "resolved": "https://registry.npmjs.org/prelude-ls/-/prelude-ls-1.2.1.tgz",
           "integrity": "sha512-vkcDPrRZo1QZLbn5RLGPpg/WmIQ65qoWWhcGKf/b5eplkkarX0m9z8ppCat4mlOqUsWpyNuYgO3VRyrYHSzX5g=="
         },
-        "prettier": {
```

Comment:
Please don't include changes to the `package-lock.json` for issues which don't include dependency changes.